DEVELOPER_GUIDE.md

```diff
@@ -74,6 +83,74 @@ This is typically created in PyCharm IDE by visiting the `Python Interpreter`, s
 `
 In order to run tests within the PyCharm IDE, ensure the `Python Integrated Tools` / `Testing` / `Default Test Runner` is set to `pytest`.
 
+## Running Workloads
+
+### Installation  
+
+Download the latest release of OpenSearch from https://opensearch.org/downloads.html. If you are using WSL, make sure to download it into your `/home/<user>` directory instead of `/mnt/c`. 
+```
+wget https://artifacts.opensearch.org/releases/bundle/opensearch/<x.x.x>/opensearch-<x.x.x>-linux-x64.tar.gz
+tar -xf opensearch-x.x.x-linux-x64.tar.gz
+cd opensearch-x.x.x
+```
+NOTE: Have Docker running in the background for the next steps. Refer to the installation instructions [here](https://docs.docker.com/compose/install/).
+
+### Setup 
+
+Add the following settings to the `opensearch.yml` file under the config directory 
+```
+vim config/opensearch.yml
+```
+```
+#
+discovery.type: single-node 
+plugins.security.disabled: true 
+#
+```
+Run the opensearch-tar-install.sh script to install and setup a cluster for our use. 
+```
+bash opensearch-tar-install.sh
+```
+Check the output of `curl.exe "http://localhost:9200/_cluster/health?pretty"`. Output should be similar to this:
+```
+{
+  "cluster_name" : "<name>",
+  "status" : "green",
+  "timed_out" : false,
+  "number_of_nodes" : 1,
+  "number_of_data_nodes" : 1,
+  "discovered_master" : true,
+  "discovered_cluster_manager" : true,
+  "active_primary_shards" : 3,
+  "active_shards" : 3,
+  "relocating_shards" : 0,
+  "initializing_shards" : 0,
+  "unassigned_shards" : 0,
+  "delayed_unassigned_shards" : 0,
+  "number_of_pending_tasks" : 0,
+  "number_of_in_flight_fetch" : 0,
+  "task_max_waiting_in_queue_millis" : 0,
+  "active_shards_percent_as_number" : 100.0
+}
+```
+Now, you have a local cluster running! You can connect to this and run the workload for the next step. 
+
+### Running the workload
```

Comment:
Would be more descriptive so that this is for running an end to end test / workload on a locally installed OpenSearch cluster